WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Hazmat Section/WooShippingHazmatRow.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct WooShippingHazmatRow: View {
+    /// Whether the interactions (navigation/setting selection) are enabled.
+    private let enabled: Bool
+
+    @Binding private var isHazardous: Bool
+
+    @State private var isShowingDetailView = false
+
+    init(isHazardous: Binding<Bool>, enabled: Bool) {
+        self._isHazardous = isHazardous
+        self.enabled = enabled
+    }
+
+    var body: some View {
+        Button(action: {
+            isShowingDetailView = true
+        }) {
+            AdaptiveStack {
+                Text(Localization.hazmatLabel)
+                    .bodyStyle()
+                Spacer()
+                Text(isHazardous ? Localization.yes : Localization.no)
+                    .secondaryBodyStyle()
+                Image(uiImage: .chevronImage)
+                    .secondaryBodyStyle()
+                    .renderedIf(enabled)
+            }
+            .padding(.vertical, Layout.verticalPadding)
+        }
+        .buttonStyle(.plain)
+        .disabled(!enabled)
+        .sheet(isPresented: $isShowingDetailView) {
+            WooShippingHazmatDetailView(isHazardous: $isHazardous)
+        }
+    }
+}
+
+private extension WooShippingHazmatRow {
+    enum Layout {
+        static let backgroundRadius: CGFloat = 8
+        static let verticalPadding: CGFloat = 24
+    }
+
+    enum Localization {
+        static let hazmatLabel = NSLocalizedString("wooShipping.createLabel.hazmatLabel",
```

Comment:
Suggestion to keep it consistent with other strings in this file. 
```suggestion
        static let hazmatLabel = NSLocalizedString("wooShipping.createLabel.hazmatRow.label",
```

Reply:
Done in 8193a5b09a7344d67551d8cecb2c057ddeed3bdd 👍